src/NzbDrone.Core/Indexers/Definitions/Avistaz/AvistazBase.cs

```diff
@@ -13,12 +13,12 @@ namespace NzbDrone.Core.Indexers.Definitions.Avistaz
     public abstract class AvistazBase : TorrentIndexerBase<AvistazSettings>
     {
         public override DownloadProtocol Protocol => DownloadProtocol.Torrent;
-        public override string[] IndexerUrls => new string[] { "" };
-        protected virtual string LoginUrl => Settings.BaseUrl + "api/v1/jackett/auth";
+        public override string[] IndexerUrls => new[] { "" };
```

Comment:
Can we drop this override from the base here to force the need to create it on all derived classes.